Rubberduck.RibbonDispatcher/AbstractRibbonDispatcher/Interfaces.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
```

Comment:
This file should probably be a namespace; avoid multiple types per source file.

Reply:
Yes - though there is an argument for having interfaces in their own assembly to avoid crossed dependencies. I don't think this functionality is big enough on its own to need that, unless Rubberduck already does that elsewhere.